client/my-sites/site-settings/spam-filtering-settings.jsx

```diff
@@ -59,19 +58,16 @@ const SpamFilteringSettings = ( {
 		className,
 		header = null;
 
-	if (
-		! shouldShowOfferResetFlow() &&
-		! inTransition &&
-		! ( hasAkismetFeature || hasAntiSpam ) &&
-		! isValidKey
-	) {
+	if ( ! inTransition && ! ( hasAkismetFeature || hasAntiSpam ) && ! isValidKey ) {
 		return (
 			<UpsellNudge
-				description={ translate( 'Automatically remove spam from comments and contact forms.' ) }
+				description={ translate(
+					'Save time, get more responses, give your visitors a better experience - all without lifting a finger.'
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **24** times:
`translate( 'Automatically clear spam from comments and forms. Save time, get more responses, give your visitors a better experience – all without lifting a finger.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/improve/anti-spam-upsell-copy)